src/renderers/shaders/ShaderChunk/lights_pars.glsl

```diff
@@ -138,8 +140,16 @@
 		float dotNL = dot( geometry.normal, hemiLight.direction );
 		float hemiDiffuseWeight = 0.5 * dotNL + 0.5;
 
+	#if defined( PHYSICALLY_CORRECT_LIGHTS )
+
+		return mix( hemiLight.groundColor, hemiLight.skyColor, hemiDiffuseWeight );
+
+	#else
+
 		return PI * mix( hemiLight.groundColor, hemiLight.skyColor, hemiDiffuseWeight );
```

Comment:
I've refactor the code to remove repetition in the PHYSICAL_CORRECT_LIGHTS true and false case, although I stayed away from the LIGHTS_PI define in this update as I find that a define that changes between PI and 1.0 could potentially be confusing.  I think that until @WestLangley or someone else validates that the Physically Correct Lights mode is really physical, I do not want to obscure what it is doing too much.